eng/pipelines/templates/steps/install-pipeline-generation.yml

```diff
@@ -1,8 +1,11 @@
+parameters:
+  ToolPath: $(Pipeline.Workspace)/pipeline-generator
+
 steps:
   - script: >
       dotnet tool install
       Azure.Sdk.Tools.PipelineGenerator
       --version 1.0.2-dev.20200502.1
       --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk/nuget/v3/index.json
-      --global
+      --tool-path ${{parameters.ToolPath}}
```

Comment:
@weshaggard you've probably already figured out why I did this, but just in case you are still wondering, this was to overcome some of the reliability issues we've been seeing with global tools installs.